Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-api/2.12.1-SNAPSHOT/jspwiki-api-2.12.1-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-api/2.11.0/jspwiki-api-2.11.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.core.Acl  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Enumeration<org.apache.wiki.api.core.AclEntry><org.apache.wiki.api.core.AclEntry> aclEntries()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean addEntry(org.apache.wiki.api.core.AclEntry)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.security.Principal[] findPrincipals(java.security.Permission)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.AclEntry getAclEntry(java.security.Principal)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isEmpty()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean removeEntry(org.apache.wiki.api.core.AclEntry)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.core.AclEntry  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean addPermission(java.security.Permission)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean checkPermission(java.security.Permission)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.security.Principal getPrincipal()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Enumeration<java.security.Permission><java.security.Permission> permissions()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean removePermission(java.security.Permission)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean setPrincipal(java.security.Principal)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.core.Attachment  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Cloneable
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: org.apache.wiki.api.core.Page
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getFileName()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getParentName()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isCacheable()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setCacheable(boolean)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setFileName(java.lang.String)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.core.Command  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getContentTemplate()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getJSP()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getName()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getRequestContext()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object getTarget()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getURLPattern()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.security.Permission requiredPermission()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Command targetedCommand(java.lang.Object)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.core.Context  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Cloneable
	===  UNCHANGED INTERFACE: org.apache.wiki.api.core.Command
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String VAR_WYSIWYG_EDITOR_MODE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String VAR_EXECUTE_PLUGINS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ATTR_CONTEXT
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Context clone()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Context deepClone()
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.api.core.Context findContext(javax.servlet.jsp.PageContext)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean getBooleanWikiProperty(java.lang.String, boolean)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Command getCommand()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.security.Principal getCurrentUser()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Engine getEngine()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getHttpParameter(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT javax.servlet.http.HttpServletRequest getHttpRequest()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Page getPage()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Page getRealPage()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getRedirectURL()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getTemplate()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getURL(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.String getURL(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object getVariable(java.lang.String)
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getViewURL(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Session getWikiSession()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean hasAdminPermissions()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setPage(org.apache.wiki.api.core.Page)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Page setRealPage(org.apache.wiki.api.core.Page)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setRequestContext(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setTemplate(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setVariable(java.lang.String, java.lang.Object)
===  UNCHANGED ENUM: PUBLIC FINAL org.apache.wiki.api.core.ContextEnum  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: java.lang.Enum (<- java.lang.Enum)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_PREFS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_ADMIN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_CREATE_GROUP
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_INSTALL
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum GROUP_VIEW
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_INFO
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_ATTACH
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_VIEW
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_UPLOAD
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_COMMENT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_DELETE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_WORKFLOW
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_NONE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_FIND
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_PREVIEW
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_RSS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_LOGIN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_DIFF
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_EDIT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_RENAME
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum GROUP_DELETE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum PAGE_CONFLICT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_MESSAGE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum GROUP_EDIT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum REDIRECT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_ERROR
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL org.apache.wiki.api.core.ContextEnum WIKI_LOGOUT
	===  UNCHANGED METHOD: PUBLIC java.lang.String getContentTemplate()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getRequestContext()
	===  UNCHANGED METHOD: PUBLIC java.lang.String getUrlPattern()
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.api.core.ContextEnum valueOf(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.api.core.ContextEnum[] values()
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.core.Engine  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_NO_FILTER_ENCODING
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_URLCONSTRUCTOR
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_SEARCHPATH
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_INTERWIKIREF
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_ENCODING
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_MATCHPLURALS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_FRONTPAGE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PREFS_COOKIE_NAME
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_TEMPLATEDIR
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_ACL_MANAGER_IMPL
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_WORKDIR
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_APPNAME
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String DEFAULT_TEMPLATE_NAME
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_STOREUSERNAME
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_ALLOW_CREATION_OF_EMPTY_PAGES
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String DEFAULT_INLINEPATTERN
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_INLINEIMAGEPTRN
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String PROP_REF_MANAGER_IMPL
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Engine adapt(java.lang.Class<E><E>)
		GENERIC TEMPLATES: === E:org.apache.wiki.api.core.Engine
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void addWikiEventListener(org.apache.wiki.event.WikiEventListener)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String decodeName(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String encodeName(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.net.URL findConfigFile(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<java.lang.String><java.lang.String> getAllInlinedImagePatterns()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<java.lang.String><java.lang.String> getAllInterWikiLinks()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getApplicationName()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object getAttribute(java.lang.String)
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getBaseURL()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.nio.charset.Charset getContentEncoding()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getFinalPageName(java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getFrontPage()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getGlobalRSSURL()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getInterWikiURL(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object getManager(java.lang.Class<T><T>)
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<T><T> getManagers(java.lang.Class<T><T>)
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC java.lang.String getPluginSearchPath()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getRootPath()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT javax.servlet.ServletContext getServletContext()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getSpecialPageReference(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Date getStartTime()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getTemplateDir()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getURL(java.lang.String, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Properties getWikiProperties()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getWorkDir()
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) void initialize(java.util.Properties)
		+++  NEW EXCEPTION: org.apache.wiki.api.exceptions.WikiException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isConfigured()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object removeAttribute(java.lang.String)
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void removeWikiEventListener(org.apache.wiki.event.WikiEventListener)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setAttribute(java.lang.String, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void shutdown()
	+++! NEW METHOD: PUBLIC(+) void start(java.util.Properties)
		+++  NEW EXCEPTION: org.apache.wiki.api.exceptions.WikiException
	+++! NEW METHOD: PUBLIC(+) void stop()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.core.Page  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.lang.Cloneable
	===  UNCHANGED INTERFACE: java.lang.Comparable
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CHANGENOTE
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String DESCRIPTION
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String VIEWCOUNT
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ALIAS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String AUTHOR
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String REDIRECT
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Page clone()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Acl getAcl()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object getAttribute(java.lang.String)
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Map<java.lang.String,java.lang.Object><java.lang.String,java.lang.Object> getAttributes()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getAuthor()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Date getLastModified()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getName()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT long getSize()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getVersion()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getWiki()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean hasMetadata()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void invalidateMetadata()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.Object removeAttribute(java.lang.String)
		GENERIC TEMPLATES: === T:java.lang.Object
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setAcl(org.apache.wiki.api.core.Acl)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setAttribute(java.lang.String, java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setAuthor(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setHasMetadata()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setLastModified(java.util.Date)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setSize(long)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void setVersion(int)
**** MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.core.Session  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ANONYMOUS
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ASSERTED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String AUTHENTICATED
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void addMessage(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void addMessage(java.lang.String, java.lang.String)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String antiCsrfToken()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void clearMessages()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void clearMessages(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.Object doPrivileged(org.apache.wiki.api.core.Session, java.security.PrivilegedAction<?><?>)
		===  UNCHANGED EXCEPTION: java.security.AccessControlException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Locale getLocale()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.security.Principal getLoginPrincipal()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String[] getMessages()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String[] getMessages(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.security.Principal[] getPrincipals()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.security.Principal[] getRoles()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getStatus()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT javax.security.auth.Subject getSubject()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.security.Principal getUserPrincipal()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean hasPrincipal(java.security.Principal)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void invalidate()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isAnonymous()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isAsserted()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean isAuthenticated()
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) org.apache.wiki.api.engine.EngineLifecycleExtension  (not serializable)
	+++  CLASS FILE FORMAT VERSION: 55.0 <- n.a.
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) void onInit(java.util.Properties)
	+++  NEW METHOD: PUBLIC(+) void onShutdown(org.apache.wiki.api.core.Engine, java.util.Properties)
	+++  NEW METHOD: PUBLIC(+) void onStart(org.apache.wiki.api.core.Engine, java.util.Properties)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.engine.Initializable  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.WikiException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.engine.RenderApi  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String textToHTML(org.apache.wiki.api.core.Context, java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.exceptions.FilterException  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.api.exceptions.WikiException (<- org.apache.wiki.api.exceptions.WikiException)
	===  UNCHANGED CONSTRUCTOR: PUBLIC FilterException(java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.exceptions.NoRequiredPropertyException  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.api.exceptions.WikiException (<- org.apache.wiki.api.exceptions.WikiException)
	===  UNCHANGED CONSTRUCTOR: PUBLIC NoRequiredPropertyException(java.lang.String, java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.exceptions.NoSuchVariableException  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.api.exceptions.WikiException (<- org.apache.wiki.api.exceptions.WikiException)
	===  UNCHANGED CONSTRUCTOR: PUBLIC NoSuchVariableException(java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.exceptions.PluginException  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.api.exceptions.WikiException (<- org.apache.wiki.api.exceptions.WikiException)
	===  UNCHANGED CONSTRUCTOR: PUBLIC PluginException(java.lang.String)
	===  UNCHANGED CONSTRUCTOR: PUBLIC PluginException(java.lang.String, java.lang.Throwable)
	===  UNCHANGED METHOD: PUBLIC java.lang.Throwable getRootThrowable()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.exceptions.ProviderException  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.api.exceptions.WikiException (<- org.apache.wiki.api.exceptions.WikiException)
	===  UNCHANGED CONSTRUCTOR: PUBLIC ProviderException(java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.exceptions.RedirectException  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.api.exceptions.FilterException (<- org.apache.wiki.api.exceptions.FilterException)
	===  UNCHANGED CONSTRUCTOR: PUBLIC RedirectException(java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC java.lang.String getRedirect()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.exceptions.WikiException  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: java.lang.Exception (<- java.lang.Exception)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiException(java.lang.String)
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiException(java.lang.String, java.lang.Throwable)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.exceptions.WikiI18nException  (compatible)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: java.io.Serializable
	===  UNCHANGED SUPERCLASS: org.apache.wiki.api.exceptions.WikiException (<- org.apache.wiki.api.exceptions.WikiException)
	===  UNCHANGED FIELD: PROTECTED FINAL java.lang.Object[] args
	===  UNCHANGED CONSTRUCTOR: PUBLIC WikiI18nException(java.lang.String, java.lang.Object[])
	===  UNCHANGED METHOD: PUBLIC java.lang.Object[] getArgs()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.filters.BasePageFilter  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.api.filters.PageFilter
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PROTECTED org.apache.wiki.api.core.Engine m_engine
	===  UNCHANGED CONSTRUCTOR: PUBLIC BasePageFilter()
	===  UNCHANGED METHOD: PUBLIC void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.filters.PageFilter  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC void destroy(org.apache.wiki.api.core.Engine)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC void postSave(org.apache.wiki.api.core.Context, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String postTranslate(org.apache.wiki.api.core.Context, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String preSave(org.apache.wiki.api.core.Context, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
	===  UNCHANGED METHOD: PUBLIC java.lang.String preTranslate(org.apache.wiki.api.core.Context, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.FilterException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.plugin.InitializablePlugin  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void initialize(org.apache.wiki.api.core.Engine)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.PluginException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.plugin.ParserStagePlugin  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void executeParser(org.apache.wiki.api.plugin.PluginElement, org.apache.wiki.api.core.Context, java.util.Map<java.lang.String,java.lang.String><java.lang.String,java.lang.String>)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.plugin.Plugin  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String CORE_PLUGINS_RESOURCEBUNDLE
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String execute(org.apache.wiki.api.core.Context, java.util.Map<java.lang.String,java.lang.String><java.lang.String,java.lang.String>)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.PluginException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.plugin.PluginElement  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void executeParse(org.apache.wiki.api.core.Context)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.PluginException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getParameter(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Map<java.lang.String,java.lang.String><java.lang.String,java.lang.String> getParameters()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getPluginName()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getText()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getValue()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String invoke(org.apache.wiki.api.core.Context)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.providers.AttachmentProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.api.providers.WikiProvider
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String PROP_STORAGEDIR
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void deleteAttachment(org.apache.wiki.api.core.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void deleteVersion(org.apache.wiki.api.core.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<org.apache.wiki.api.core.Attachment><org.apache.wiki.api.core.Attachment> findAttachments(org.apache.wiki.api.search.QueryItem[])
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.io.InputStream getAttachmentData(org.apache.wiki.api.core.Attachment)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Attachment getAttachmentInfo(org.apache.wiki.api.core.Page, java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<org.apache.wiki.api.core.Attachment><org.apache.wiki.api.core.Attachment> getVersionHistory(org.apache.wiki.api.core.Attachment)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<org.apache.wiki.api.core.Attachment><org.apache.wiki.api.core.Attachment> listAllChanged(java.util.Date)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<org.apache.wiki.api.core.Attachment><org.apache.wiki.api.core.Attachment> listAttachments(org.apache.wiki.api.core.Page)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void moveAttachmentsForPage(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void putAttachmentData(org.apache.wiki.api.core.Attachment, java.io.InputStream)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
		===  UNCHANGED EXCEPTION: java.io.IOException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.providers.PageProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED INTERFACE: org.apache.wiki.api.providers.WikiProvider
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void deletePage(java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void deleteVersion(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<org.apache.wiki.api.search.SearchResult><org.apache.wiki.api.search.SearchResult> findPages(org.apache.wiki.api.search.QueryItem[])
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<org.apache.wiki.api.core.Page><org.apache.wiki.api.core.Page> getAllChangedSince(java.util.Date)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.Collection<org.apache.wiki.api.core.Page><org.apache.wiki.api.core.Page> getAllPages()
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getPageCount()
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Page getPageInfo(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getPageText(java.lang.String, int)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.util.List<org.apache.wiki.api.core.Page><org.apache.wiki.api.core.Page> getVersionHistory(java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void movePage(java.lang.String, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean pageExists(java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT boolean pageExists(java.lang.String, int)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void putPageText(org.apache.wiki.api.core.Page, java.lang.String)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.ProviderException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.providers.WikiProvider  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int LATEST_VERSION
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String getProviderInfo()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void initialize(org.apache.wiki.api.core.Engine, java.util.Properties)
		===  UNCHANGED EXCEPTION: org.apache.wiki.api.exceptions.NoRequiredPropertyException
		===  UNCHANGED EXCEPTION: java.io.IOException
===  UNCHANGED CLASS: PUBLIC FINAL org.apache.wiki.api.Release  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int REVISION
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String BUILD
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String APPNAME
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int VERSION
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String VERSTR
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int MINORREVISION
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String getVersionString()
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isNewerOrEqual(java.lang.String)
		===  UNCHANGED EXCEPTION: java.lang.IllegalArgumentException
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isOlderOrEqual(java.lang.String)
		===  UNCHANGED EXCEPTION: java.lang.IllegalArgumentException
	===  UNCHANGED METHOD: PUBLIC STATIC void main(java.lang.String[])
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.search.QueryItem  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int REQUIRED
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int REQUESTED
	===  UNCHANGED FIELD: PUBLIC int type
	===  UNCHANGED FIELD: PUBLIC java.lang.String word
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int FORBIDDEN
	===  UNCHANGED CONSTRUCTOR: PUBLIC QueryItem()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.search.SearchResult  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT java.lang.String[] getContexts()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Page getPage()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT int getScore()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.spi.AclsDSL  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Acl acl()
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.AclEntry entry()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.spi.AclsSPI  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Acl acl()
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.AclEntry entry()
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.spi.ContentsDSL  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Attachment attachment(org.apache.wiki.api.core.Engine, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Page page(org.apache.wiki.api.core.Engine, java.lang.String)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.spi.ContentsSPI  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Attachment attachment(org.apache.wiki.api.core.Engine, java.lang.String, java.lang.String)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Page page(org.apache.wiki.api.core.Engine, java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.spi.ContextDSL  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Context create(org.apache.wiki.api.core.Engine, org.apache.wiki.api.core.Page)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Context create(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest, org.apache.wiki.api.core.Command)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Context create(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest, org.apache.wiki.api.core.Page)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Context create(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest, java.lang.String)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.spi.ContextSPI  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Context create(org.apache.wiki.api.core.Engine, org.apache.wiki.api.core.Page)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Context create(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest, org.apache.wiki.api.core.Command)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Context create(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest, org.apache.wiki.api.core.Page)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Context create(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest, java.lang.String)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.spi.EngineDSL  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Engine find(javax.servlet.ServletConfig)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Engine find(javax.servlet.ServletConfig, java.util.Properties)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Engine find(javax.servlet.ServletContext, java.util.Properties)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.spi.EngineSPI  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Engine find(javax.servlet.ServletContext, java.util.Properties)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.spi.SessionDSL  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Session find(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest)
	===  UNCHANGED METHOD: PUBLIC org.apache.wiki.api.core.Session guest(org.apache.wiki.api.core.Engine)
	===  UNCHANGED METHOD: PUBLIC void remove(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest)
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.wiki.api.spi.SessionSPI  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Session find(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.wiki.api.core.Session guest(org.apache.wiki.api.core.Engine)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT void remove(org.apache.wiki.api.core.Engine, javax.servlet.http.HttpServletRequest)
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.api.spi.Wiki  (not serializable)
	***! CLASS FILE FORMAT VERSION: 55.0 <- 52.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED CONSTRUCTOR: PUBLIC Wiki()
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.api.spi.AclsDSL acls()
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.api.spi.ContentsDSL contents()
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.api.spi.ContextDSL context()
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.api.spi.EngineDSL engine()
	===  UNCHANGED METHOD: PUBLIC STATIC java.util.Properties init(javax.servlet.ServletContext)
	===  UNCHANGED METHOD: PUBLIC STATIC org.apache.wiki.api.spi.SessionDSL session()
